Broncode/operatiebrp-code-145.3/algemeen/alg-dal-repositories-jpa/src/main/java/nl/bzk/algemeenbrp/dal/repositories/jpa/BasisRepositoryImpl.java

```diff
@@ -50,9 +50,9 @@ protected BasisRepositoryImpl(final Class<T> entityType) {
     protected final EntityManager getEntityManager() { return entityManager; }
 
     /**
-     * Geef de waarde van entityManager.
+     * Zet de waarde van entityManager.
      *
-     * @return entityManager
+     * @param EntityManager entityManager
```

Comment:
Geachte,

In tegenstelling tot bij PHP, dient het type van een parameter niet te worden opgenomen in het documentie blok.

In afwachting van uw correctie.

Reply:
Dit is volgens de standaard gehanteerd in de rest van de programmatuur, zie bijvoorbeeld regel 32.

Reply:
` @param {@link EntityManager entityManager}`

Reply:
Even of-topic: ik verwacht geen acceptatie van de PR of enige activiteit anders dan het uploaden van de source hier. Ik verwacht veel meer van een fork waar men zich op kan uitleven (denk bijvoorbeeld aan het packagen van wat er is en eventueel door ontwikkelen). Wie lijkt dit wat?

Reply:
@kwoot wat kan er nog doorontwikkelt worden? Waar zijn de requirements. Is er een roadmap?

Misschien is het een idee om die documentatie directory om te zetten naar iets dat toegankelijker is, zodat nieuwe contributors een beter beeld hebben waar dit over gaat?

Reply:
Goeie vraag. Ik denk als we er in eerste instantie in slagen om een via packages installeerbaar product te maken (BPR-server) we al een heel eind zijn. Ik verwacht zelf dat deze dan dankbaar gebruik gaat worden om add-ons voor te ontwikkelen. Doc dir omzetten naar iets toegankelijkers lijkt me een goed idee. Wat stel je voor quoa docformat?

Reply:
Aanvullend verzoek ingediend: https://twitter.com/JeroenBaten/status/936180340728778752